src/test/fixtures/profiles/processed-profile.js

```diff
@@ -1457,10 +1461,11 @@ export function getProfileWithBalancedNativeAllocations() {
     { byteSize: 11, stack: E, memoryAddress: 0x20 },
     { byteSize: 13, stack: Gjs, memoryAddress: 0x21 },
     { byteSize: 17, stack: I, memoryAddress: 0x22 },
-    // Deallocations that match the first group.
-    { byteSize: -3, stack: E, memoryAddress: 0x10 },
-    { byteSize: -5, stack: Gjs, memoryAddress: 0x11 },
-    { byteSize: -7, stack: I, memoryAddress: 0x12 },
+    // Deallocations that match the first group. Deallocations don't always
+    // happen at the same stack.
+    { byteSize: -3, stack: J, memoryAddress: 0x10 },
+    { byteSize: -5, stack: K, memoryAddress: 0x11 },
+    { byteSize: -7, stack: Gjs, memoryAddress: 0x12 },
```

Comment:
Same thing than before here: I take care that the stacks are different than the allocations, so that we know that only memoryAddress is used to match.